sphinx/texinputs/Makefile_t

```diff
@@ -57,30 +58,18 @@ bz2: tar
 xz: tar
 	xz -9 -k $(ARCHIVEPREFIX)docs-$(FMT).tar
 
-# The number of LaTeX runs is quite conservative, but I don't expect it
-# to get run often, so the little extra time won't hurt.
-%.dvi: %.tex
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-	-$(MAKEINDEX) -s python.ist '$(basename $<).idx'
-	$(LATEX) $(LATEXOPTS) '$<'
+%.dvi: %.tex FORCE_MAKE
 	$(LATEX) $(LATEXOPTS) '$<'
 
-%.pdf: %.tex
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	-$(MAKEINDEX) -s python.ist '$(basename $<).idx'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
+%.pdf: %.tex FORCE_MAKE
 	$(PDFLATEX) $(LATEXOPTS) '$<'
```

Comment:
This is very useful to me thanks !

I mention in passing that Latexmk by default uses `"` quotes around strings from its place holders like `%O`, `%S`. Thus on Linux we must be carefult not to had one more  `"` layer. But this is something else than the Make variables.